pkg/generators/pipelines/fluentd/templates.go

```diff
@@ -39,6 +39,12 @@ const fluentConfTemplate = `{{- define "fluentConf" }}
 	@include configs.d/openshift/filter-k8s-record-transform.conf
 	@include configs.d/openshift/filter-syslog-record-transform.conf
 	@include configs.d/openshift/filter-viaq-data-model.conf
+	<filter **>
```

Comment:
you can't just `@include configs.d/openshift/filter-post-genid.conf` ?

Reply:
We could but we should be providing the entire config instead of relying on a mix match between conf generation and static includes.